testing/tests/create-election.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { createElection, API_BASE_URL } from './helperfunctions';
+import { test, expect, Page } from '@playwright/test';
+
+let electionId = '';
+test.describe('Create Election', () => {
+    test('create poll', async ({ page }) => {
+        page.goto('/');
+        await page.getByRole('button', { name: 'New Election' }).click();
+        await page.getByLabel('Poll', { exact: true }).click();
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.fill('input[name="election-title"]', 'Playwright Test Poll');
+        //wait until there is only one continue button
+        while ((await page.getByRole('button', { name: 'Continue' }).evaluateAll((el) => el)).length > 1) {
+        continue
+        }
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.getByLabel('No').click();
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.getByText('Allows multiple votes per device').click();
+        await expect(page.getByText('draft')).toBeVisible();
+        const url = await page.url();
+        const urlArray = url.split('/');
+        electionId = urlArray[urlArray.length - 2];
+        expect(await page.getByLabel('no limit')).toBeChecked();
+
+    });
+
+    test('create election with email list', async ({ page }) => {
+        await page.goto('/');
+        await page.getByRole('button', { name: 'New Election' }).click();
+        await page.getByLabel('Election', { exact: true }).click();
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.fill('input[name="election-title"]', 'Playwright Test Election');
+        //wait until there is only one continue button
+        while ((await page.getByRole('button', { name: 'Continue' }).evaluateAll((el) => el)).length > 1) {
+        continue
+        }
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.getByLabel('Yes').click();
+        await page.getByRole('button', { name: 'Continue' }).click();
+        await page.getByRole('button', { name: 'Email List Provide a list of' }).click();
+        await expect(page.getByText('draft')).toBeVisible();
+        const url = await page.url();
```

Comment:
Yeah, that would probably be good. I had a hard time getting helper functions to work outside of tests though because the await keyword means something else within a test than it normally does within an async function